src/Views/Page.elm

```diff
@@ -293,35 +295,49 @@ versionLink version =
 
 
 pageHeader : Config msg -> Html msg
-pageHeader config =
+pageHeader { session, activePage, openMobileNavigation, loadUrl, switchVersion } =
     header [ class "Header shadow-sm", attribute "role" "banner" ]
         [ div [ class "MobileMenuButton" ]
             [ button
                 [ type_ "button"
                 , class "d-inline-block d-sm-none btn m-0 p-0"
                 , attribute "aria-label" "Ouvrir la navigation"
                 , title "Ouvrir la navigation"
-                , onClick config.openMobileNavigation
+                , onClick openMobileNavigation
                 ]
                 [ span [ class "fs-3" ] [ Icon.ham ] ]
             ]
-        , Container.centered [ class "d-flex justify-content-between align-items-center" ]
+        , Container.centered [ class "d-flex justify-content-between align-items-center gap-2" ]
             [ a
                 [ class "HeaderBrand text-decoration-none d-flex align-items-center gap-3 gap-sm-5 pe-3"
 
                 -- Note: this class makes Dashlord understand DSFR guidelines are implemented
                 -- https://dashlord.mte.incubateur.net/dashlord/url/ecobalyse-beta-gouv-fr/best-practices/#dsfr
                 , class "fr-header__brand"
                 , href "/"
+                , onClick (loadUrl "/")
                 ]
                 [ img [ class "HeaderLogo", alt "République Française", src "img/republique-francaise.svg" ] []
                 , h1 [ class "HeaderTitle" ] [ text "Ecobalyse" ]
                 ]
+            , session.releases
+                |> RemoteData.map
+                    (List.map
+                        (\release ->
+                            option [ selected <| Version.is release session.currentVersion ]
+                                [ text release.tag ]
+                        )
+                    )
```

Comment:
What if the current version is not found on github? I know it can't happen in production but it's a little bit strange when developing locally we have the `v2.0.0` selected by default. In this case, maybe we could add a "Unreleased" entry or something along the lines?

Reply:
@vjousse addressed in https://github.com/MTES-MCT/ecobalyse/pull/700/commits/8673fab287d9858391f105b897721dc7a6e6db9b

Do you want me to patch the injected js as well?

Reply:
I don't think that it makes sense as we are only patching released versions.